src/app/shared/money-input/money-input.component.spec.ts

```diff
@@ -0,0 +1,27 @@
+/* tslint:disable:no-unused-variable */
```

Comment:
Remove that rule as well as all variables you're not using

Reply:
done